api/src/deployment.rs

```diff
@@ -365,8 +364,7 @@ impl DeploymentSystem {
     async fn initialise_from_fs(fs_root: &Path) -> Deployments {
         let mut deployments = HashMap::default();
         for project_dir in std::fs::read_dir(fs_root)
-            .unwrap() // safety: api can read the fs root dir
-            .into_iter() {
+            .unwrap() {
```

Comment:
could still be not safe - fs root probably exists, but it doesn't mean the app can read it